src/uct/sm/mm/sysv/mm_sysv.c

```diff
@@ -59,7 +59,8 @@ static ucs_status_t uct_sysv_mem_attach_common(int shmid, void **address_p)
 
 static ucs_status_t
 uct_sysv_mem_alloc(uct_md_h tl_md, size_t *length_p, void **address_p,
-                   unsigned flags, const char *alloc_name, uct_mem_h *memh_p)
+                   ucs_memory_type_t mem_type, unsigned flags,
```

Comment:
in all md's mem alloc, add check that it's a right mem type

Reply:
I've added a generic check in uct_md_mem_alloc implementation instead.